src/share/classes/com/sun/javatest/regtest/tool/ExcludeFileVerifier.java

```diff
@@ -0,0 +1,118 @@
+package com.sun.javatest.regtest.tool;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.util.regex.Pattern;
+import java.util.function.Predicate;
+
+public class ExcludeFileVerifier {
+    private static String testName(String line) {
+        line = line.trim();
+        String[] words = line.split("\\s+");
+        return words.length >= 1 ? words[0] : null;
+    }
+
+    private boolean hadErrors = false;
+    public boolean getHadErrors() {
+        return hadErrors;
+    }
+    public boolean verify(File file, List<String> validTestNames) {
+        ArrayList<String> usedTestNames = new ArrayList<String>();
+        ArrayList<Check> checks = new ArrayList<Check>();
+        checks.add(new LineFormatCheck());
+        checks.add(new TestExistsCheck(validTestNames));
+        checks.add(new DuplicateCheck(usedTestNames));
+
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(file));
+            String line = null;
+            int n = 0;
+            while ((line = br.readLine()) != null)
+            {
+                n++;
+                if (lineIsComment(line.trim())) continue;
+                for(Check c : checks) {
+                    if(!c.check(line.trim())) {
+                        System.out.println(file.getAbsolutePath() + " line " + n + " is invalid. Reason:");
+                        System.out.println(c.description());
+                        System.out.println("Line contents:");
+                        System.out.println("--------------");
+                        System.out.println(line);
+                        System.out.println("--------------");
```

Comment:
Don't use `System.out` directly like this; pass in a `PrintWriter` to be used. 

The main `jtreg``Tool` class sets up two stream to be used, `out` and `err`.  
Generally, 
* `standard output` / `System.out` / `sysOut` / `stdOut` should be used for the expected output of specific options, like command-line help.
* `standard error` / `System.err` / `sysErr` / `stdErr` should be used for diagnostic output when issues are found performing any requested operation.